core/trino-main/src/main/java/io/trino/execution/QueryManagerConfig.java

```diff
@@ -617,19 +616,6 @@ public QueryManagerConfig setFaultTolerantExecutionPartitionCount(int faultToler
         return this;
     }
 
-    public boolean getFaultTolerantPreserveInputPartitionsInWriteStage()
-    {
-        return faultTolerantPreserveInputPartitionsInWriteStage;
-    }
-
-    @Config("fault-tolerant-execution-preserve-input-partitions-in-write-stage")
-    @ConfigDescription("Ensure single task reads single hash partitioned input partition for stages which write table data")
```

Comment:
should we just change default and deprecate config property in case someone uses that?

Reply:
This property was never documented and in reality there's practically no reason of changing the defaults.  I doubt it has ever been used by anyone. I would prefer to remove this property to avoid complicating the partition assignment algorithm with extra branches (that would also require extra testing).